test_collect-logs.py

```diff
@@ -360,6 +365,27 @@ def test_success(self):
                                       stderr=subprocess.STDOUT,
                                       env=None,
                                       ))
+        # for _create_ps_mem_output_file()
+        for host in self.hosts:
```

Comment:
Since our unit test is only setting up a single host, maybe we can explicitly check that there is only one mocked juju ssh "0", cmd called.